test/python/transpiler/test_high_level_synthesis.py

```diff
@@ -534,22 +534,22 @@ def test_section_size(self):
             hls_config = HLSConfig(linear_function=[("pmh", {"section_size": 1})])
             qct = HighLevelSynthesis(hls_config=hls_config)(qc)
             self.assertEqual(LinearFunction(qct), LinearFunction(qc))
-            self.assertEqual(qct.size(), 22)
-            self.assertEqual(qct.depth(), 20)
+            self.assertEqual(qct.size(), 30)
```

Comment:
Do these change due to a different random matrix? Did you check by chance that with the old Python implementation of `random_invertible_binary_matrix` the results are still the same?

Reply:
these tests are based on fixed seed, and so the numbers were changed since the random function is different in Python and Rust